include/envoy/router/router_ratelimit.h

```diff
@@ -69,6 +69,11 @@ class RateLimitPolicy {
   virtual ~RateLimitPolicy() {}
 
   /**
+   * @return true if there is no rate limit policy.
+   */
+  virtual bool empty() const PURE;
```

Comment:
Oh I see, we can't exactly, because stage() isn't known until "run time". So we need to know if it's fully empty. I might just briefly expand the comment on why we need this.